### ISSUE TITLE
[feature] Dart support

### ISSUE THREAD

**Issue by itsezc** [moonrepo/moon#749]
**Is your feature request related to a problem? Please describe.**

Support for the Dart language and ecosystem, so that one could use moon for Flutter based apps

**Comment by nyomansunima** [comment]
Yeah, this feature may need to be added. Sometimes we need to develop on other stacks. I explored most of the mono repo tools but did not find one that suit what I need. Something like flexible and easy setup

**Comment by milesj** [comment]
@nyomansunima You can technically run any language/system command from moon. Integrated language support just provides a few more benefits.

**Comment by lalitkumawat1m** [comment]
Hey, you are doing great work. I want to contribute to this project how can I start?

**Comment by listepo** [comment]
Any news?

**Comment by milesj** [comment]
New languages are paused until we move over to plugins.

**Comment by listepo** [comment]
any news?

**Comment by milesj** [comment]
Now that toolchain plugins are available, this can be built by the community. Don't plan on supporting this officially in moon.

**Comment by tuga001-sme** [comment]
I think official support is needed, Flutter is very popular now.

**Comment by milesj** [comment]
We have absolutely no bandwidth to support every language officially. We built a plugin system so that the community can implement these themselves.

**Comment by listepo** [comment]
@milesj Has anything changed in priorities and bandwidth? And if this is added as a pending request, can it be accepted?

**Comment by milesj** [comment]
Since Dart isn't as popular as other languages, this will need to be contributed by the community.

Here's an example of someone implementing .NET: https://github.com/moonrepo/plugins/pull/169
